dev/release/verify-release-candidate.sh

```diff
@@ -649,15 +651,17 @@ test_and_install_cpp() {
   export CMAKE_BUILD_PARALLEL_LEVEL=${CMAKE_BUILD_PARALLEL_LEVEL:-${NPROC}}
   cmake --build . --target install
 
-  # Explicitly set site-package directory, otherwise the C++ tests are unable
-  # to load numpy in a python virtualenv
-  local pythonpath=$(python -c "import site; print(site.getsitepackages()[0])")
-
-  LD_LIBRARY_PATH=$PWD/release:$LD_LIBRARY_PATH PYTHONPATH=$pythonpath ctest \
-    --label-regex unittest \
-    --output-on-failure \
-    --parallel $NPROC \
-    --timeout 300
+  if [ ${TEST_CPP} -gt 0 ]; then
```

Comment:
Can we also disable `ARROW_BUILD_TESTS` otherwise?

Reply:
Ah, we can do it.